modules/troubleshooting-network-observability-loki-large-query-timeout.adoc

```diff
@@ -0,0 +1,24 @@
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-troubleshooting-large-query-timeout_{context}"]
+= Running a large query results in a Loki timeout error
```

Comment:
```suggestion
= Running a large query results in a Loki error, such as a timeout or "too many outstanding requests"
```

Reply:
As I said in the comment over there, I think instead of adding this to the title, can we adjust the introduction to the Troubleshooting:
> When running large queries for a long time, Loki errors can occur, such as a timeout or too many outstanding requests.

Reply:
and update the title to be instead:
= Running a large query results in Loki errors

Reply:
sgtm!